src/quemb/kbe/fragment.py

```diff
@@ -74,6 +77,7 @@ def fragmentate(
     frozen_core=False,
     self_match=False,
     allcen=True,
+    print_frags=True,
```

Comment:
I would either forward the `print_frags` argument also to `autogen` or not expose it.